web/src/main/scala/net/wiringbits/components/widgets/SignInForm.scala

```diff
@@ -95,10 +96,28 @@ import scala.util.{Failure, Success}
         )
     )
 
-    val error = formData.firstValidationError.map { text =>
+    def resendVerifyEmailButton(text: String): ReactElement = {
+      val emailNotVerified = ErrorMessages.emailNotVerified
+
+      text match {
+        case `emailNotVerified` =>
```

Comment:
Use this instead `ErrorMessages.emailNotVerified` which makes it obvious that we are referencing an error message.

Also, let's add a To Do, like `// TODO: It would be ideal to match the error against a code than matching a text`